lib/analysis/author/graph/generate.py

```diff
@@ -70,8 +74,8 @@ def author_interaction():
 					json_data[json_obj['Message-ID']] = json_obj
 		print("JSON data loaded.")
 	else:
-		lone_author_threads = get_lone_author_threads(False)
-		with open('clean_data.json', 'r') as json_file:
+		lone_author_threads = get_lone_author_threads(nodelist_filename=graph_nodes, edgelist_filename=graph_edges)
```

Comment:
please use positional arguments here.

Reply:
`get_lone_author_threads()` is a function of read.py, which was to be refactored afterward.